package-lock.json

```diff
@@ -10784,7 +10784,7 @@
     },
     "ora": {
       "version": "0.2.3",
-      "resolved": "http://registry.npmjs.org/ora/-/ora-0.2.3.tgz",
+      "resolved": "https://registry.npmjs.org/ora/-/ora-0.2.3.tgz",
```

Comment:
I accidentally pushed some changes to the package-lock.json that I didn't mean to. This is to fix it. Sorry for any confusion!